src/ScheduledTasks/Client/ScheduledTaskClientImpl.cs

```diff
@@ -0,0 +1,182 @@
+            ScheduleOperationRequest request = new ScheduleOperationRequest(entityId, nameof(Schedule.CreateSchedule), creationOptions);
+            string instanceId = await this.durableTaskClient.ScheduleNewOrchestrationInstanceAsync(
+                new TaskName(nameof(ExecuteScheduleOperationOrchestrator)),
+                request,
+                cancellation);
+
+            // Wait for the orchestration to complete
+            OrchestrationMetadata state = await this.durableTaskClient.WaitForInstanceCompletionAsync(instanceId, true, cancellation);
+
+            if (state.RuntimeStatus != OrchestrationRuntimeStatus.Completed)
+            {
+                throw new InvalidOperationException($"Failed to create schedule '{scheduleId}': {state.FailureDetails?.ErrorMessage ?? string.Empty}");
+            }
+
+            // Return a handle to the schedule
+            return new ScheduleClientImpl(this.durableTaskClient, scheduleId, this.logger);
+        }
+        catch (OperationCanceledException ex)
+        {
+            this.logger.ClientError(
+                nameof(this.CreateScheduleAsync),
+                creationOptions.ScheduleId,
+                ex);
+
+            throw new OperationCanceledException(
+                $"The {nameof(this.CreateScheduleAsync)} operation was canceled.",
+                null,
+                cancellation);
+        }
+    }
+
+    /// <inheritdoc/>
+    public override async Task<ScheduleDescription?> GetScheduleAsync(string scheduleId, CancellationToken cancellation = default)
+    {
+        Check.NotNullOrEmpty(scheduleId, nameof(scheduleId));
+
+        try
+        {
+            EntityInstanceId entityId = new EntityInstanceId(nameof(Schedule), scheduleId);
+            EntityMetadata<ScheduleState>? metadata = await this.durableTaskClient.Entities.GetEntityAsync<ScheduleState>(entityId, cancellation);
+
+            if (metadata == null || metadata.State.Status == ScheduleStatus.Uninitialized)
+            {
+                return null;
+            }
+
+            ScheduleState state = metadata.State;
+            ScheduleConfiguration? config = state.ScheduleConfiguration;
+
+            return new ScheduleDescription
+            {
+                ScheduleId = scheduleId,
+                OrchestrationName = config?.OrchestrationName,
+                OrchestrationInput = config?.OrchestrationInput,
+                OrchestrationInstanceId = config?.OrchestrationInstanceId,
+                StartAt = config?.StartAt,
+                EndAt = config?.EndAt,
+                Interval = config?.Interval,
+                StartImmediatelyIfLate = config?.StartImmediatelyIfLate,
+                Status = state.Status,
+                ExecutionToken = state.ExecutionToken,
+                LastRunAt = state.LastRunAt,
+                NextRunAt = state.NextRunAt,
+            };
+        }
+        catch (OperationCanceledException ex)
```

Comment:
This exception handler seems not quite right:
(a) when operations are cancelled normally we need not log it as an error
(b) we should be logging all other exceptions

Also, I don't think it is necessary to wrap exceptions.

Here is sort of what I would expect to see:
```csharp
catch (OperationCanceledException) when (cancellation.IsCancellationRequested)
{
    // the operation was cancelled as requested. No need to log this.
    throw;
}
catch (Exception ex)
{
    this.logger.ClientError(
        nameof(this.GetScheduleAsync),
        scheduleId,
        ex);

    throw;
}
```